src/components/Filters/components/ConnectedFilterControl/ConnectedFilterControl.scss

```diff
@@ -67,7 +68,7 @@ $stacking-order: (
 .RightContainer {
   display: flex;
 
-  .Item > * > * {
+  .Item > div > button {
```

Comment:
Are these always rendered in a button group? I wonder if we could use our segmented data attribute here i.e.
```
.Item > [data-buttongroup-segmented='true'] {
  .Button {}
}
```

Reply:
Looks like we never actually use the Polaris `ButtonGroup` component in `Filters`.

We create individual `Button`s using the `activatorButtonFrom` function on `Line 234` in `ConnectedFilterControl.tsx` and then use CSS to override the border-radius / spacing

Reply:
I think it would make sense to use a `ButtonGroup` so that we can delete a lot of the CSS in here.
I can try to tackle that in a future PR 👍